examples/gamma-control.c

```diff
@@ -115,21 +115,26 @@ static void fill_gamma_table(uint16_t *table, uint32_t ramp_size,
 	uint16_t *b = table + 2 * ramp_size;
 	for (uint32_t i = 0; i < ramp_size; ++i) {
 		double val = (double)i / (ramp_size - 1);
-		val = contrast * pow(val, 1.0 / gamma) + brightness;
+		val = contrast * pow(val, 1.0 / gamma) + (1 - brightness);
```

Comment:
```(brightness - 1)``` would be better. 
In my opinion ```brightness = 0``` should result in a black image, ```2``` pure white.

Reply:
Oh right